### PR TITLE
Single genome build search

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: ui/pages/Report/components/CustomSearch.jsx [hunk 1]
```diff
@@ -43,8 +43,11 @@ const PROJECT_FAMILIES_FIELD = {
 
 const INCLUDE_ALL_PROJECTS_FIELD = {
   name: INCLUDE_ALL_PROJECTS,
-  component: InlineToggle,
-  fullHeight: true,
+  component: ButtonRadioGroup,
+  options: [
+    ...GENOME_VERSION_OPTIONS.map(opt => ({ ...opt, color: 'black' })),
+    { value: '', text: 'Custom', color: 'grey' },
```
Comment (ShifaSZ) [inline ui/pages/Report/components/CustomSearch.jsx:49]: `Custom` is obscure. How about `Single Project Search` or others?

**Review by ShifaSZ — CHANGES_REQUESTED** [review]
Mostly looks very good to me.

File: ui/pages/Report/components/CustomSearch.jsx [hunk 2]
```diff
@@ -15,11 +15,11 @@ import { AddProjectButton, ProjectFilter } from 'shared/components/panel/search/
 import VariantSearchResults from 'shared/components/panel/search/VariantSearchResults'
 import DataLoader from 'shared/components/DataLoader'
 import { InlineHeader } from 'shared/components/StyledComponents'
-import { INHERITANCE_FILTER_OPTIONS, ALL_INHERITANCE_FILTER } from 'shared/utils/constants'
+import { INHERITANCE_FILTER_OPTIONS, ALL_INHERITANCE_FILTER, GENOME_VERSION_OPTIONS } from 'shared/utils/constants'
 import { loadProjectContext, loadProjectGroupContext, loadSearchHashContext } from '../reducers'
 import { getSearchHashContextLoading } from '../selectors'
 
-const INCLUDE_ALL_PROJECTS = 'allProjectFamilies'
+const INCLUDE_ALL_PROJECTS = 'allGenomeProjectFamilies'
```
[the remark this answers is not in the document]
Comment (hanars) [reply ui/pages/Report/components/CustomSearch.jsx:22]: No

**Review by ShifaSZ — APPROVED** [review]
Excellent work!

**Pull request merged**